core/server/master/src/main/java/alluxio/master/file/FileSystemMasterAuditContext.java

```diff
@@ -37,6 +37,7 @@ public final class FileSystemMasterAuditContext implements AuditContext {
   private Inode mSrcInode;
   private long mCreationTimeNs;
   private long mExecutionTimeNs;
+  private String mClientVersion;
```

Comment:
make this nullable

Reply:
There are many properties that will be nullable here, this is just to keep same with the previous.